qiskit/circuit/quantumcircuit.py

```diff
@@ -3483,6 +3464,13 @@ def depth(
         ),
     ) -> int:
         """Return circuit depth (i.e., length of critical path).
+        The depth of a quantum circuit is a measure of how many
```

Comment:
Formatters won't like the lack of space

```suggestion
        """Return circuit depth (i.e., length of critical path).

        The depth of a quantum circuit is a measure of how many
```